.infrastructure/deployment.yml

```diff
@@ -52,3 +52,20 @@ spec:
             port: 8080
           initialDelaySeconds: 5
           periodSeconds: 5
+        volumeMounts:
+          - name: todoapp-secrets-volume
+            mountPath: /app/secrets
+            readOnly: true
+          - name: todoapp-config-volume
+            mountPath: /app/configs
+            readOnly: true
+      volume:
```

Comment:
The key 'volume' should be 'volumes' to correctly define the list of volumes in the pod specification.